src/index.js

```diff
@@ -0,0 +1,28 @@
+const identity = (value) => value;
+
+const nullReducer = (state) => typeof state === 'undefined' ? null : state;
+
+/**
+ * Creates a higher-order reducer which map actions before passing to a reducer
+ * Specify a list of action types to whitelist actions for mapping.
+ *
+ * mapAction(
+ *   actionMapper: ?((action: Object) => newAction: Object) = (x) => x,
+ *   actionTypes: ?Array<string>
+ * ): (reducer) => reducer
+ */
+const mapAction = (actionMapper = identity, actionTypes = null) => {
```

Comment:
I think `actionMapper` should be required, because this HOR useless without it :)